stackit/internal/services/opensearch/credential/resource.go

```diff
@@ -157,12 +155,6 @@ func (r *credentialResource) Schema(_ context.Context, _ resource.SchemaRequest,
 				ElementType: types.StringType,
```

Comment:
Opensearch credentials has a new string field `Scheme`, can you add it?

Reply:
added new field